### ISSUE TITLE
ipc.send from browser-laptop doesn't work with Uint8Arrays

### ISSUE THREAD

**Issue by diracdeltas** [brave/sync#17]
if the message contains a Uint8Array, it never gets sent for other reasons. sending Uint8Arrays from webview to browser-laptop works fine though.

**Comment by bridiver** [comment]
not sure I understand. ipcRenderer.send works, but not webContents.send? That seems very odd because the same converters will apply in both directions. Where are you trying to send from?

**Comment by diracdeltas** [comment]
@bridiver no, ipcMain.send (ex: in browser-laptop app/index.js) doesn't work with Uint8Arrays

**Comment by bridiver** [comment]
ipcMain.send doesn't go anywhere. What are you trying to send to?

**Comment by diracdeltas** [comment]
my bad, i meant evt.sender.send in an ipcMain.on handler. see https://github.com/brave/browser-laptop/blob/feature/syncing/app/sync.js#L38 for instance; if you change any of the sent params to a Uint8Array, the send doesn't seem to happen.
